src/windows/Button.tsx

```diff
@@ -0,0 +1,211 @@
+/**
+* Button.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* RN Windows-specific implementation of the cross-platform Button abstraction.
+*/
+
+import React = require('react');
+import {Button as ButtonBase} from '../native-common/Button';
+import EventHelpers from '../native-common/utils/EventHelpers';
+import UserInterface from '../native-desktop/UserInterface';
+import RN = require('react-native');
+import RNW = require('react-native-windows');
+import { applyFocusableComponentMixin, FocusManagerFocusableComponent } from '../native-desktop/utils/FocusManager';
+
+const KEY_CODE_ENTER = 13;
+const KEY_CODE_SPACE = 32;
+
+const DOWN_KEYCODES = [KEY_CODE_SPACE, KEY_CODE_ENTER];
+const UP_KEYCODES = [KEY_CODE_SPACE];
+
+let _isNavigatingWithKeyboard = false;
+
+UserInterface.keyboardNavigationEvent.subscribe(isNavigatingWithKeyboard => {
+   _isNavigatingWithKeyboard = isNavigatingWithKeyboard;
+});
+
+// Simple check for the presence of the updated React Native for Windows
+const IsUpdatedReactNativeForWindows = (RNW.FocusableWindows !== undefined);
```

Comment:
>IsUpdatedReactNativeForWindows [](start = 6, length = 30)

Consider calling this "HasFocusableWindows" or something similar. I think it's better to think in terms of the APIs the platform supports rather than thinking in terms of versions.

Reply:
ok

---
In reply to: [159336672](https://github.com/Microsoft/reactxp/pull/404#discussion_r159336672) [](ancestors = 159336672)